ord_schema/message_helpers_test.py

```diff
@@ -132,8 +132,9 @@ def test_get_reaction_smiles(self):
             value='Cc1ccccc1', type='SMILES')
         reactant2.components.add(reaction_role='SOLVENT').identifiers.add(
             value='N', type='SMILES')
-        reaction.outcomes.add().products.add().identifiers.add(value='O=C=O',
-                                                               type='SMILES')
+        reaction.outcomes.add().products.add(
+            reaction_role=reaction_pb2.ReactionRole.PRODUCT).identifiers.add(
```

Comment:
For future reference, you can write `reaction_role='PRODUCT'` instead of `reaction_role=reaction_pb2.ReactionRole.PRODUCT`

Reply:
Great, thanks. Wasn't sure. Saw this construct elsewhere and used it here to be safe. Updated!